bounter/count_min_sketch.py

```diff
@@ -7,53 +7,43 @@
 # This code is distributed under the terms and conditions
 # from the MIT License (MIT).
 
-import random
-from HLL import HyperLogLog
-
-import numpy
-import xxhash
-
-
-def hash_key(row, key):
-    """
-    Calculates a hash for a key for a specific row of the table
-    The row is used as a seed in order to minimize correlations between collisions
-    :return hash of the key as int
-    """
-    return xxhash.xxh64(str(key), seed=row).intdigest()
+import cmsc
 
 
 class CountMinSketch(object):
     """
     Data structure used to estimate frequencies of elements in massive data sets with fixed memory footprint.
-
     Example::
-
         cms = CountMinSketch(size_mb=512) # Use 512 MB
         print(cms.width)  # 16 777 216
```

Comment:
As a convention `>>>` is used for code examples in docstrings:
```
>>> cms = CountMinSketch(size_mb=512)
>>> print(cms.depth)
```

The bonus for doing so is that Pycharm recognizes the convention and checks the code for you.